### PR TITLE
Add VIA support to DZ60

### PR DESCRIPTION
**Pull request by minajevs**
## Description
Adds VIA support keymap to `dz60` board. Tested on my current DZ60 build and it works perfect. 

[Related VIA PR](https://github.com/the-via/keyboards/pull/19)

## Types of Changes
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR
* N/A

## Checklist
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dz60/config.h [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x4b46  // KF as if KbdFans
```
Comment (noroadsleft) [inline keyboards/dz60/config.h]:
```suggestion
#define VENDOR_ID       0x4B46  // "KF" as if KbdFans
```

File: keyboards/dz60/keymaps/via/rules.mk [hunk 2]
```diff
@@ -0,0 +1,3 @@
+MOUSEKEY_ENABLE = no
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/via/rules.mk]: Does enabling VIA without the Mouse Keys and LTO changes make the firmware too large to flash when compiled?

**Comment by noroadsleft** [comment]
The Org60 and remote tracking merges prompted me to look into the state of your fork.

If you follow the instructions in the following docs, you should be able to sync your fork more easily:

- [Resynchronizing a Branch - QMK Firmware Docs](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch)
- [Using Your Fork's Master - QMK Firmware Docs](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

If you need any help with this, let me know.

File: keyboards/dz60/keymaps/via/rules.mk (hunk 2 above)
Comment (minajevs) [reply keyboards/dz60/keymaps/via/rules.mk]: > The Org60 and remote tracking merges prompted me to look into the state of your fork.
> 
> If you follow the instructions in the following docs, you should be able to sync your fork more easily:
> 
> * [Resynchronizing a Branch - QMK Firmware Docs](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch)
> * [Using Your Fork's Master - QMK Firmware Docs](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)
> 
> If you need any help with this, let me know.

Thanks, I noticed that way too late. Should I recreate a PR with a fresh state? I can't seem to fix that at this point without recreating

**Comment by minajevs** [comment]
@noroadsleft 
Thanks, I noticed that way too late. Should I recreate a PR with a fresh state? I can't seem to fix that at this point without recreating, unless you can help me with that

File: keyboards/dz60/keymaps/via/rules.mk (hunk 2 above)
Comment (minajevs) [reply keyboards/dz60/keymaps/via/rules.mk]: Those rules are VIA keymap specific, so it should. I've also verified locally that there are same build results with and without them.

**Pull request closed without merging**